File: src/main/java/me/miki/shindo/management/security/SecurityFeatureManager.java
package me.miki.shindo.management.security;

import me.miki.shindo.management.security.impl.*;

import java.util.ArrayList;

public class SecurityFeatureManager {

    private final ArrayList<SecurityFeature> features = new ArrayList<SecurityFeature>();

    public SecurityFeatureManager() {
        features.add(new DemoSecurity());
        features.add(new ExplosionSecurity());
        features.add(new Log4jSecurity());
        features.add(new ParticleSecurity());
        features.add(new ResourcePackSecurity());
        features.add(new TeleportSecurity());
    }

    public ArrayList<SecurityFeature> getFeatures() {
        return features;
    }
}

File: src/main/kotlin/me/miki/shindo/management/security/SecurityFeatureManager.kt
package me.miki.shindo.management.security

import me.miki.shindo.management.security.impl.DemoSecurity
import me.miki.shindo.management.security.impl.ExplosionSecurity
import me.miki.shindo.management.security.impl.Log4jSecurity
import me.miki.shindo.management.security.impl.ParticleSecurity
import me.miki.shindo.management.security.impl.ResourcePackSecurity
import me.miki.shindo.management.security.impl.TeleportSecurity

class SecurityFeatureManager {
    private val features = ArrayList<SecurityFeature>()

    init {
        features.add(DemoSecurity())
        features.add(ExplosionSecurity())
        features.add(Log4jSecurity())
        features.add(ParticleSecurity())
        features.add(ResourcePackSecurity())
        features.add(TeleportSecurity())
    }

    fun getFeatures(): ArrayList<SecurityFeature> = features
}
